src/generateTrackerData.ts

```diff
@@ -0,0 +1,60 @@
+import { IAnalyticsTrackerOptions } from './types';
+
+export const generateTrackerData = (
+  options: IAnalyticsTrackerOptions,
+): { metricName: string; duration?: number; value?: any } => {
+  const { metricName, data } = options;
```

Comment:
`navigatorInformation` has some important Device Information that helps understand where the metric is coming from, let's make sure to have it there as well.